firewalldb/actions_test.go

```diff
@@ -440,6 +504,9 @@ func TestListGroupActions(t *testing.T) {
 }
 
 func assertEqualActions(t *testing.T, expected, got *Action) {
+	// Accounts are not explicitly linked in our bbolt DB implementation.
+	got.AccountID = expected.AccountID
```

Comment:
nit: Should we remove the overwritten value from `got` at the end of this function as this is a pointer we're passing?

Reply:
sure